File: 3st/Hyeonseo/programmer/배열 뒤집기.cpp
#include <string>
#include <vector>

using namespace std;

vector<int> solution(vector<int> num_list) {
        vector<int> answer(num_list.size());
        for (int i = 0; i < num_list.size(); i++) {
            answer[i] = num_list[num_list.size() - i - 1];
        }
        return answer;
    }

File: 4st/Seohyun/programmer/배열 뒤집기.java
class Solution {
    public int[] solution(int[] num_list) {
        
        int size=num_list.length;
        int[] answer=new int[size];
        
        for (int i=0; i<size; i++) {
            answer[i]=num_list[size-(i+1)];
        }
        return answer;
    }
}